contracts/core/exchange-wrappers/ZeroExExchangeWrapper.sol

```diff
@@ -94,22 +94,21 @@ contract ZeroExExchangeWrapper {
         address[] memory takerTokens = new address[](_orderCount);
         uint256[] memory takerAmounts = new uint256[](_orderCount);
 
-        // First 32 bytes are reserved for the number of orders
-        // uint256 orderNum = 0;
-        // uint256 offset = 32;
-        // while (offset < _orderData.length) {
-        //     bytes memory zeroExOrder = OrderHandler.sliceOrderBody(_orderData, offset);
+        uint256 orderNum = 0;
+        uint256 offset = 0;
```

Comment:
Does this work without a 32 byte offset?

Reply:
Merging in first, I plan to clean up `getZeroExOrderDataLength` and `sliceOrderBody` and i'll get to the bottom of it.